app.py

```diff
@@ -49,8 +49,29 @@ def _get_local_time():
         except Exception as e:
             return f"Could not process time information. {e}", 404
 
+@app.route('/time', methods=['GET'])
+def all_time():
+    timezones = ['America/Los+Angeles', 'America/Denver', 'America/Chicago', 'America/New+York']
```

Comment:
I noticed that pytz has a us timezone list available, might be more helpful than a hard-coded list.